upf/upf.c

```diff
@@ -382,6 +389,10 @@ upf_init (vlib_main_t * vm)
   sm->upf_simple_counters[UPF_FLOW_COUNTER].stat_segment_name =
     "/upf/total_flows";
 
+  sm->node_id.type = NID_FQDN;
+  u8 *name = 0;
+  sm->node_id.fqdn = format (name, (char *) "upg");
```

Comment:
Type FQDN needs a DNS label encoded value, not a string like the initialization is using.

```suggestion
  sm->node_id.type = NID_FQDN;
  sm->node_id.fqdn = format (0, "\x03upg");
```

Reply:
Done